src/Kanvas/Companies/Observers/CompaniesObserver.php

```diff
@@ -92,6 +93,7 @@ public function created(Companies $company): void
                 'company' => $company,
             ]
         );
+        (new Setup(app(Apps::class), $user, $company))->run();
```

Comment:
@FredPeal no lets have a workflow or another observer where we create all the setups for company, we have it on onboarding